part2.asm

```diff
@@ -0,0 +1,114 @@
+# COMP3410
+# Author: Daniel Thompson
+# Assignment: PA[1] Part 2
+# Date: 2/12/15
+
+#######################################
+# .data segment
+#######################################
+.data
```

Comment:
Be careful in how you declare variables in the .data section.
It appears that declaring some strings, then some words, then other strings can cause problems. You should put them all together by type.

Your program was printing odd things and this was why